koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -591,30 +366,13 @@ def handler(self, src, target_info, arch, output_template, scratch=False,
         containerdata = {
             'arch': arch,
             'task_id': self.id,
-            'logs': logs,
             'osbs_build_id': build_id,
-            'rpmlist': rpmlist,
             'files': [],
             'repositories': repositories
         }
 
-        # upload the build output
-        for filename in containerdata['logs']:
-            build_log = os.path.join(self.workdir, filename)
-            self.uploadFile(build_log)
-
-        if response.is_succeeded() and len(files) != 1:
-            self.logger.error("There should be only one container file but "
-                              "there are %d: %s", len(files), files)
-        else:
-            for filename in files:
-                full_path = os.path.join(self.workdir, filename)
-                self.uploadFile(full_path, remoteName=output_template)
-                containerdata['files'].append(os.path.basename(output_template))
-
         return containerdata
 
-
```

Comment:
This is a stupid thing, but PEP8 says we should have two blank lines between classes. Won't hold up the merge for it though.